### ISSUE TITLE
Pyflakes and pep8 raise many concerns

### ISSUE THREAD

**Issue by jkahn** [EducationalTestingService/rsmtool#27]
A few minor nitpicks from pyflakes
<details>
```
$ pyflakes3 rsmtool/*.py
rsmtool/__init__.py:11: 'rsmextra' imported but unused
rsmtool/model.py:508: local variable 'intercept' is assigned to but never used
rsmtool/predict.py:38: local variable 'logger' is assigned to but never used
rsmtool/rsmpredict.py:320: local variable 'logger' is assigned to but never used
```
</details>
and *lots* of complaints (about line length, whitespace, boolean conditions, etc) from pep8.  Flagged results from many files:
<details>
```
$ pep8 -q rsmtool/
rsmtool/analysis.py
rsmtool/create_features.py
rsmtool/input.py
rsmtool/model.py
rsmtool/predict.py
rsmtool/preprocess.py
rsmtool/report.py
rsmtool/rsmcompare.py
rsmtool/rsmeval.py
rsmtool/rsmpredict.py
rsmtool/rsmtool.py
rsmtool/test_utils.py
rsmtool/utils.py
```
</details>

**Comment by desilinguist** [comment]
Thanks! Some of these are probably helpful.

I usually write Python code with the AutoPEP8 plugin enabled but with the
length constraint turned off because I think that's a little silly in
today's day and age (see Raymond Hettinger's really nice talk on PEP-8:
https://m.youtube.com/watch?v=wf-BqAjZb8M)

**Comment by jkahn** [comment]
I tried it while ignoring the length constraint and still got dozens of warnings, so: do what you like. If I were you I'd run a linter-&c check as part of your CI tests, turning off whatever warnings you don't like.

**Comment by desilinguist** [comment]
That's a good idea. I'll take a look.

**Comment by desilinguist** [comment]
Addressed by #39 and #40.
